.github/workflows/main.yml

```diff
@@ -0,0 +1,116 @@
+name: Alfresco tomcat base Docker image
+on:
+  push:
+    branches:
+      #TODO add master when confident
+      - opsexp-[0-9]+**
+
+env:
+  IMAGE_REGISTRY_NAMESPACE: alxgomz
```

Comment:
Do we want to use your nickname as registry namespace?

Reply:
oops. fixed